js/h5p.js

```diff
@@ -224,10 +224,16 @@ H5P.init = function (target) {
         instance.getCurrentState instanceof Function ||
         typeof instance.getCurrentState === 'function')) {
 
-      var saveTimer, save = function () {
+      var saveTimer, lastSavedState, save = function () {
         var state = instance.getCurrentState();
-        if (state !== undefined) {
+
+        //compare last state with current state
+        var stateChanged = H5P.eq(state, lastSavedState, null, null);
```

Comment:
This variable is named very confusingly.  `stateChanged` is only true when the state has not changed?